legacylogin/src/main/java/com/hamdan/forzenbook/legacy/login/LegacyLoginActivity.kt

```diff
@@ -0,0 +1,141 @@
+package com.hamdan.forzenbook.legacy.login
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.viewModels
+import androidx.core.view.isVisible
+import androidx.core.widget.addTextChangedListener
+import androidx.lifecycle.viewModelScope
+import com.hamdan.forzenbook.core.LoginError
+import com.hamdan.forzenbook.legacy.core.view.utils.DialogUtils.standardDialog
+import com.hamdan.forzenbook.legacy.core.viewmodels.LegacyLoginViewModel
+import com.hamdan.forzenbook.legacy.login.databinding.ActivityLegacyLoginBinding
+import com.hamdan.forzenbook.ui.core.R
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+@AndroidEntryPoint
+class LegacyLoginActivity : ComponentActivity() {
+    // TODO FA-104 add a click listener to send to Create Account on the createAccount text
+    private lateinit var binding: ActivityLegacyLoginBinding
+    private val loginModel: LegacyLoginViewModel by viewModels()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityLegacyLoginBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
+        binding.apply {
+            var emailInputTextValue = ""
+            var codeInputTextValue = ""
+            loginSubmitButton.isEnabled = false
+            loginClickBlocker.setOnClickListener { }
```

Comment:
TODO or remove

Reply:
it can't be removed, it needs to exist to prevent clicks on items

Reply:
put a comment.